shared/types/Tournament/index.d.ts

```diff
@@ -24,28 +24,30 @@ export interface MatchScore {
 
 export interface Match {
   id?: string;
-  teamA?: Team;
-  teamB?: Team;
+  teamA: Team | null;
+  teamB: Team | null;
   childMatchAId?: string;
   childMatchBId?: string;
   score: MatchScore;
   isFinished: boolean;
 }
 
 export interface MatchWithoutMS extends Omit<Match, "teamA" | "teamB"> {
-  teamA?: TeamWithoutMS;
-  teamB?: TeamWithoutMS;
+  teamA: TeamWithoutMS | null;
+  teamB: TeamWithoutMS | null;
   childTeamsAmount?: number;
 }
 
+export type TournamentType = "round-robin" | "single-elimination";
```

Comment:
You should use this `TournamentType` to set `type`'s type in Tournament class

Reply:
ok

Reply:
but how?

Reply:
/server/concepts/Tournament.ts:27
```javascript
- type: 'single-elimination' | 'round-robin';
+ type: TournamentType;
```

Reply:
pushed